dev/side-nav.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Side nav</title>
+    <script type="module" src="./common.js"></script>
+    <script type="module">
+      import '@vaadin/icon';
+      import '@vaadin/icons';
+      import '@vaadin/scroller';
+      import "@vaadin/side-nav/enable";
+      import '@vaadin/side-nav';
+    </script>
+    <style>
+      vaadin-scroller {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+  <vaadin-scroller>
```

Comment:
Please format this file using Prettier and commit changes to the indentation:

```sh
$ npx prettier dev/side-nav.html --write
```

Reply:
Done.